# Timed keypoint to ROS message

float32 x
float32 y
float32 size
float32 angle
float32 response
int32 octave
int32 class_id
int64 timestamp
